src/neo/Ledger/Blockchain.cs

```diff
@@ -365,16 +383,15 @@ private RelayResultReason OnNewBlock(Block block)
                     UpdateCurrentSnapshot();
                 }
             }
-            return RelayResultReason.Succeed;
+            return VerifyResult.Succeed;
         }
 
-        private RelayResultReason OnNewConsensus(ConsensusPayload payload)
+        private VerifyResult OnNewConsensus(ConsensusPayload payload)
```

Comment:
Would it be more clear if we create another VerifyPayload?
The current way is simpler and more abstract.

Reply:
I don't understand, what is the `VerifyPayload`?

Reply:
It can be resolved, I understood it later that the VerifyResult was generic for any blocks, payloads, txs..

Perhaps the name is good.